cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -41,6 +41,7 @@
 #include <memory>
 #include <numeric>
 #include <type_traits>
+#include "thrust/functional.h"
```

Comment:
I hate when vscode does this.

Reply:
Yea - especially when it is just because you are trying something out and then you don't need it later. Would be really nice if we could get [IWYU](https://include-what-you-use.org/) set up.

Reply:
TODO fix this.